### PR TITLE
vendor runc rc92

### PR DESCRIPTION
**Pull request by tao12345666333**
* vendor containerd to v1.4.0-rc.1
    full diff https://github.com/containerd/containerd/compare/779ef60231a555f7eb9ba82b052d59b69ca2ef10...v1.4.0-rc.1

* vendor runc to v1.0.0-rc92
   full diff https://github.com/opencontainers/runc/compare/67169a9d43456ff0d5ae12b967acb8e366e2f181...ff819c7e9184c13b7c2607fe6c30ae19403a7aff
* https://github.com/golang/crypto/compare/2aa609cf4a9d7d1126360de73b55b6002f9e052a...75b288015ac94e66e3d6715fb68a9b41bf046ec2
* https://github.com/golang/net/compare/0de0cce0169b09b364e001f108dc0399ea8630b3...ab34263943818b32f575efc978a3d24e80b04bd7
* https://github.com/golang/sys/compare/9dae0f8f577553e0f21298e18926efc9644c281d...ed371f2e16b4b305ee99df548828de367527b76b

### CODE REVIEW

**Review by AkihiroSuda — APPROVED** [review]

File: vendor.conf [hunk 1]
```diff
@@ -83,10 +83,11 @@ google.golang.org/grpc                              f495f5b15ae7ccda3b38c53a1bfc
 # the containerd project first, and update both after that is merged.
 # This commit does not need to match RUNC_COMMIT as it is used for helper
 # packages but should be newer or equal.
-github.com/opencontainers/runc                      67169a9d43456ff0d5ae12b967acb8e366e2f181 # v1.0.0-rc91-48-g67169a9d
-github.com/opencontainers/runtime-spec              237cc4f519e2e8f9b235bacccfa8ef5a84df2875 # v1.0.3-0.20200520003142-237cc4f519e2
+github.com/opencontainers/runc                      ff819c7e9184c13b7c2607fe6c30ae19403a7aff # v1.0.0-rc92
+github.com/opencontainers/runtime-spec              4d89ac9fbff6c455f46a5bb59c6b1bb7184a5e43 # v1.0.3-0.20200728170252-4d89ac9fbff6
 github.com/opencontainers/image-spec                d60099175f88c47cd379c4738d158884749ed235 # v1.0.1
 github.com/seccomp/libseccomp-golang                689e3c1541a84461afc49c1c87352a6cedf72e9c # v0.9.1
+github.com/cyphar/filepath-securejoin               a261ee33d7a517f054effbf451841abaafe3e0fd # v0.2.2
```
Comment (tao12345666333) [inline vendor.conf:90]: introduced by https://github.com/opencontainers/runc/commit/a73ce38d16aecffd7eff605784e29282876646d4

File: vendor.conf [hunk 2]
```diff
@@ -140,7 +141,7 @@ github.com/golang/protobuf                          84668698ea25b64748563aa20726
 github.com/cloudflare/cfssl                         5d63dbd981b5c408effbb58c442d54761ff94fbd # 1.3.2
 github.com/fernet/fernet-go                         9eac43b88a5efb8651d24de9b68e87567e029736
 github.com/google/certificate-transparency-go       37a384cd035e722ea46e55029093e26687138edf # v1.0.20
-golang.org/x/crypto                                 2aa609cf4a9d7d1126360de73b55b6002f9e052a
+golang.org/x/crypto                                 75b288015ac94e66e3d6715fb68a9b41bf046ec2
```
Comment (tao12345666333) [inline vendor.conf:144]: just for match containerd's dependencies

File: vendor/golang.org/x/sys/unix/zsysnum_linux_amd64.go [hunk 3]
```diff
@@ -353,4 +353,6 @@ const (
 	SYS_FSPICK                 = 433
 	SYS_PIDFD_OPEN             = 434
 	SYS_CLONE3                 = 435
+	SYS_OPENAT2                = 437
+	SYS_PIDFD_GETFD            = 438
```
Comment (thaJeztah) [inline vendor/golang.org/x/sys/unix/zsysnum_linux_amd64.go:357]: SYS_OPENAT2 looks related to https://patchwork.kernel.org/patch/11167585/

- should we have these vendor-bumps in a separate commit? (are they needed for 19.03.x?)
- do we need to add `openat2` to our seccomp profile? https://github.com/moby/moby/blob/0f41a77c6993ade605a639fb25994cfe5e1b3fe8/profiles/seccomp/seccomp_default.go#L224

Comment (tao12345666333) [reply vendor/golang.org/x/sys/unix/zsysnum_linux_amd64.go:357]: Thanks, let me take a look.

Comment (tao12345666333) [reply vendor/golang.org/x/sys/unix/zsysnum_linux_amd64.go:357]: I've separated these commits. 

And I will open a PR for add `openat2` to  our seccomp profile.

**Review by AkihiroSuda — APPROVED** [review]

File: vendor/golang.org/x/sys/unix/zsysnum_linux_amd64.go (hunk 3 above)
Comment (tao12345666333) [reply vendor/golang.org/x/sys/unix/zsysnum_linux_amd64.go:357]: Opened #41353

**Review by thaJeztah — APPROVED** [review]
LGTM if green, thanks!

**Comment by tao12345666333** [comment]
```
[2020-08-14T16:08:59.782Z] === Failed
[2020-08-14T16:08:59.782Z] === FAIL: amd64.integration-cli TestDockerSuite/TestLogsFollowSlowStdoutConsumer (1.40s)
[2020-08-14T16:08:59.782Z]     --- FAIL: TestDockerSuite/TestLogsFollowSlowStdoutConsumer (1.40s)
[2020-08-14T16:08:59.782Z]         docker_cli_logs_test.go:243: assertion failed: error is not nil: exit status 1
[2020-08-14T16:08:59.782Z] 
[2020-08-14T16:08:59.782Z] === FAIL: amd64.integration-cli TestDockerSuite (1291.11s)
```

Flaky test #41276

**Pull request merged**